diff --git a/folly/debugging/symbolizer/BUILD.bazel b/folly/debugging/symbolizer/BUILD.bazel
index 853a9f2..d97ff3a 100644
--- a/folly/debugging/symbolizer/BUILD.bazel
+++ b/folly/debugging/symbolizer/BUILD.bazel
@@ -32 +31,0 @@ cpp_library(
-        "//folly/portability/provide:libdwarf",
@@ -33 +32,4 @@ cpp_library(
-    ],
+    ] + select({
+        "@platforms//os:windows": [],
+        "//conditions:default": ["//folly/portability/provide:libdwarf"],
+    }),
diff --git a/CMake/folly-config.h.cmake b/CMake/folly-config.h.cmake
index 3d3a828..dfefc95 100644
--- a/CMake/folly-config.h.cmake
+++ b/CMake/folly-config.h.cmake
@@ -59,0 +60,4 @@
+#ifdef _MSC_VER
+# undef FOLLY_HAVE_INT128_T
+#endif
+
diff --git a/folly/portability/BUILD.bazel b/folly/portability/BUILD.bazel
index 9c3a4b1..c522440 100644
--- a/folly/portability/BUILD.bazel
+++ b/folly/portability/BUILD.bazel
@@ -293,5 +293,6 @@ cpp_library(
     }),
     deps = [
+        ":sys_types",
         ":windows",
         "//folly:portability",
     ],
