diff --git a/folly/debugging/symbolizer/BUILD.bazel b/folly/debugging/symbolizer/BUILD.bazel
index 853a9f2..d97ff3a 100644
--- a/folly/debugging/symbolizer/BUILD.bazel
+++ b/folly/debugging/symbolizer/BUILD.bazel
@@ -32 +31,0 @@ cpp_library(
-        "//folly/portability/provide:libdwarf",
@@ -33 +32,4 @@ cpp_library(
-    ],
+    ] + select({
+        "@platforms//os:windows": [],
+        "//conditions:default": ["//folly/portability/provide:libdwarf"],
+    }),
diff --git a/CMake/folly-config.h.cmake b/CMake/folly-config.h.cmake
index 3d3a828..dfefc95 100644
--- a/CMake/folly-config.h.cmake
+++ b/CMake/folly-config.h.cmake
@@ -59,0 +60,5 @@
+#ifdef _MSC_VER
+#undef FOLLY_HAVE_INT128_T
+#define FOLLY_HAVE_INT128_T 0
+#endif
+
