diff --git a/folly/debugging/symbolizer/BUILD.bazel b/folly/debugging/symbolizer/BUILD.bazel
index 853a9f2..d97ff3a 100644
--- a/folly/debugging/symbolizer/BUILD.bazel
+++ b/folly/debugging/symbolizer/BUILD.bazel
@@ -32 +31,0 @@ cpp_library(
-        "//folly/portability/provide:libdwarf",
@@ -33 +32,4 @@ cpp_library(
-    ],
+    ] + select({
+        "@platforms//os:windows": [],
+        "//conditions:default": ["//folly/portability/provide:libdwarf"],
+    }),
