diff --git a/folly/debugging/symbolizer/BUILD.bazel b/folly/debugging/symbolizer/BUILD.bazel
index 853a9f2..d97ff3a 100644
--- a/folly/debugging/symbolizer/BUILD.bazel
+++ b/folly/debugging/symbolizer/BUILD.bazel
@@ -32 +31,0 @@ cpp_library(
-        "//folly/portability/provide:libdwarf",
@@ -33 +32,4 @@ cpp_library(
-    ],
+    ] + select({
+        "@platforms//os:windows": [],
+        "//conditions:default": ["//folly/portability/provide:libdwarf"],
+    }),
diff --git a/folly/Portability.h b/folly/Portability.h
index f36cb00..fe0184a 100644
--- a/folly/Portability.h
+++ b/folly/Portability.h
@@ -25,3 +25,8 @@
+#ifdef _MSC_VER
+#undef FOLLY_HAVE_INT128_T
+#define FOLLY_HAVE_INT128_T 0
+#endif
+
 #if defined(_MSC_VER)
 #define FOLLY_CPLUSPLUS _MSVC_LANG
 #else
diff --git a/folly/portability/BUILD.bazel b/folly/portability/BUILD.bazel
index 9c3a4b1..c522440 100644
--- a/folly/portability/BUILD.bazel
+++ b/folly/portability/BUILD.bazel
@@ -293,5 +293,6 @@ cpp_library(
     }),
     deps = [
+        ":sys_types",
         ":windows",
         "//folly:portability",
     ],
